lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -18,6 +18,11 @@ table.ccx-schedule {
     th, td {
```

Comment:
Hi @talbs, this partial gets included here: https://github.com/edx/edx-platform/blob/master/lms/static/sass/_build-course.scss#L60-L61